CMakeLists.txt

```diff
@@ -158,6 +160,11 @@ if (MACOS_USE_DEPENDENCIES)
     endif ()
 endif ()
 
+# If OS is Linux, import OpenSoundEffects and OpenMusic
+if(UNIX AND NOT APPLE)
+    set(DOWNLOAD_OPENMSX ON)
```

Comment:
Since this PR disabled `OPENSFX` and `OPENMSX`, shouldn't it have reenabled both according to this if's comment?